frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import * as classNames from 'classnames';
+import { sortable } from '@patternfly/react-table';
+import { ResourceLink, humanizeBinaryBytes } from '@console/internal/components/utils/index';
+import { getNodeRoles, K8sResourceKind } from '@console/internal/module/k8s';
+import { Table, TableRow, TableData } from '@console/internal/components/factory';
+
+const tableColumnClasses = [
+  classNames('col-md-1', 'col-sm-1', 'col-xs-1'),
+  classNames('col-md-5', 'col-sm-6', 'col-xs-8'),
+  classNames('col-md-2', 'col-sm-3', 'hidden-2'),
+  classNames('col-md-2', 'hidden-sm', 'hidden-xs'),
+  classNames('col-md-2', 'hidden-2', 'hidden-xs'),
+];
+
+const NodeTableHeader = () => {
+  return [
+    {
+      title: 'Name',
+      sortField: 'metadata.name',
+      transforms: [sortable],
+      props: { className: tableColumnClasses[1] },
+    },
+    {
+      title: 'Role',
+      props: { className: tableColumnClasses[2] },
```

Comment:
As per the UXD[1], currently name field should only be sortable. 

1- https://github.com/shirimordechay/openshift-origin-design/blob/OCS-Install-workflow/web-console/Storage/OCS/OCS%20_Installation_Workflow.md

Reply:
I guess my question is: why is that the design? The table is already sorted by name, so making the name column sortable doesn't really help. Blocking users from sorting on the other columns seems arbitrary since there's no technical reason to do that.

Reply:
cc @julienlim as this is a design question

Reply:
@gnehapk @spadgett @shirimordechay 

+1 to make all columns sortable and not to block users from doing so.

Reply:
@spadgett @julienlim I will remove the sorting now. Will send a separate PR for handling it.